elastic/kibana
# [Observability] [Exploratory view]  update v7 button styles

## Summary

Fixes elastic/uptime#319

This PR applies a fix to the exploratory view button to prevent re-layouts during animation by helping the browser optimize this animation through `will-change: transform`. This is the solution recommended by the eui team.

![chrome-capture (5)](https://user-images.githubusercontent.com/11356435/118270721-69e67a00-b48e-11eb-8ce5-3d0353005d1b.gif)

**Testing**
1. Open firefox
2. Go to User Experience and click Analyze Data
3. Hover over the data type and report type buttons
4. Ensure that hovering over the buttons does not cause the layout to update
